examples/analog_in/ai_voltage_waveform_cont_acq_with_event.py

```diff
@@ -9,24 +9,24 @@
 from nidaqmx.constants import AcquisitionType
 
 with nidaqmx.Task() as task:
-    samples = []
+    sample_size = 0
 
     def callback(task_handle, every_n_samples_event_type, number_of_samples, callback_data):
         """Callback function for reading singals."""
         print("Every N Samples callback invoked.")
-
-        samples.extend(task.read(number_of_samples_per_channel=100))
+        global sample_size
+        sample_size += len(task.read(number_of_samples_per_channel=100))
```

Comment:
Can you also pretty print the data?

Reply:
Yes, but I'm afraid the 100 of data may fill up the terminal for every triggered event, users may miss out the message "Running task. Press Enter to stop and see number of accumulated samples." Or it does not matter, users will find a way to abort it.

Reply:
It should be the same amount of prints as the non-event based example, though, right?

Reply:
Yup, you are right. For continuous acquisition examples will keep printing 100 sample data.
I will update this file later when home.

Reply:
Yeah, I **definitely** don't like not printing the data for finite. I don't mind printing the data for continuous, either, but I get the hesitance. However, examples are pretty contrived, so doing something spammy and weird in an example just to demonstrate that real-live data is actually flowing seems fine to me. If someone wants the example to stop they'll press a random button and quickly Ctrl+C. The former will stop the event example and the latter the non-event example.

Reply:
That callback is called 10 times a second, so sleeping for 1/2 second within it is a little odd. Maybe just move the initial print to before the `task.start` call and add a very small (1/10th a second?) wait before calling `task.start`? Give the user a little bit of time to read.

Reply:
wait is removed, it's no longer needed.